runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptionsTest.java

```diff
@@ -242,6 +244,7 @@ public void testDefaultGcpRegionFromEnvironment() {
       when(DefaultGcpRegionFactory.getRegionFromEnvironment()).thenReturn("us-west1");
       DataflowPipelineOptions options = PipelineOptionsFactory.as(DataflowPipelineOptions.class);
       assertEquals("us-west1", options.getRegion());
+      assertEquals("us-west1-" + Dataflow.DEFAULT_SERVICE_PATH, options.getDataflowEndpoint());
```

Comment:
`Dataflow.DEFAULT_SERVICE_PATH` seems to be empty (`""`). Is that what we want to pass as DataflowEndpoint?

`us-west1-`?

Reply:
Perhaps you want [DEFAULT_BASE_URL](https://developers.google.com/resources/api-libraries/documentation/dataflow/v1b3/java/latest/com/google/api/services/dataflow/Dataflow.html#DEFAULT_BASE_URL)